static/scss/detail/courses-in-program.scss

```diff
@@ -94,23 +94,26 @@
     font-weight: 500;
     margin: 0 0 20px;
     line-height: 30px;
-    color: black;
   }
 
   p {
     margin: 0 0 10px;
     min-height: 89px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    text-align: start;
   }
 
   .read-more {
     position: absolute;
-    bottom: 5px;
+    bottom: 20px;
+    left: 15px;
     font-size: 18px;
     line-height: 24px;
     display: inline-block;
     vertical-align: top;
```

Comment:
I think we don't need these two properties. 
```
display: inline-block;
vertical-align: top;
```

Reply:
Removed